changelogs/fragments/8783-preserve-parameters-provided-from-task-in-zypper-module.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - zypper-module - only read attributes ``enabled``, ``disable_gpg_check`` and ``autorefresh`` from local repo files, if the parameters weren't already provided by the task parameters
```

Comment:
```suggestion
  - zypper-module - only read attributes ``enabled``, ``disable_gpg_check`` and ``autorefresh`` from local repo files, if the parameters were not already provided by the task parameters (https://github.com/ansible-collections/community.general/pull/9108).
```

Reply:
Hi @TobiasZeuch181 please add this modification as well.